src/it/scala/io/iohk/ethereum/transactionTest/ContractTest.scala

```diff
@@ -0,0 +1,40 @@
+package io.iohk.ethereum.transactionTest
+
+import io.iohk.ethereum.ledger.Ledger
+import io.iohk.ethereum.transactionTest.util.FixtureProvider
+import org.scalatest.{FlatSpec, Matchers}
+
+import scala.language.postfixOps
+
+class ContractTest extends FlatSpec with Matchers {
+  "Ledger" should "transfer ether" in {
+    val fixtures: FixtureProvider.Fixture = FixtureProvider.loadFixtures("/transactionTest/purchaseContruct/")
+
+    val (storage, stateStorage) = FixtureProvider.prepareStorages(0, fixtures)
+
+    //block only with ether transfers
+    Ledger.executeBlock(fixtures.blockByNumber(1), storage, stateStorage)
+
+    //block 2 contains contract creation
+    //block 3 and 4 contains contract calls with contract storage modification
+    //block 4 contains contract paying to 2 accounts
```

Comment:
As all this blocks apart from block 4 are executed in other tests, should we remove this comments from here? Should we also add some test for block 4?